core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -171,7 +171,7 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
               (canCatalog, canShare, canCompute, owners, authDomain, bucketDetails) <- DBIO.from(futuresInParallel)
               stats <- getWorkspaceSubmissionStats(workspaceContext, dataAccess)
             } yield {
-              RequestComplete(StatusCodes.OK, WorkspaceResponse(accessLevel, canShare, canCompute, canCatalog, WorkspaceDetails(workspaceContext.workspace, authDomain.toSet), stats, bucketDetails, owners))
+              RequestComplete(StatusCodes.OK, WorkspaceResponse(Some(accessLevel), Some(canShare), Some(canCompute), Some(canCatalog), WorkspaceDetails(workspaceContext.workspace, authDomain.toSet), Some(stats), Some(bucketDetails), Some(owners)))
```

Comment:
channeling doge: use `Option()`, not `Some()`

Reply:
because of nulls? That didn't feel possible here, but I could make the change.

Reply:
yeah -- easier to say "we do it everywhere" than "we engage our brain and get it wrong sometimes"